src/style.css

```diff
@@ -1,3 +1,66 @@
 body {
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic font family in case the specific font you've specified is unavailable. For example, you can specify 'sans-serif' as a fallback for 'Roboto'. (CSS checklist)